samples/subsys/ipc/openamp_rsc_table/remote/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 3.13.1)
+# Copyright (c) 2019  STMicroelectronics
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+
+project(openamp_rsc_table_remote)
+
+zephyr_compile_definitions(
+  -DMETAL_MAX_DEVICE_REGIONS=2
+  -DVIRTIO_SLAVE_ONLY
```

Comment:
ideally we get -DVIRTIO_SLAVE_ONLY this from my PR, but fine for now.

Reply:
Please, could you point out your PR, to help me to understand your comment?

Reply:
It's this old PR https://github.com/zephyrproject-rtos/zephyr/pull/14750 which I plan on updating.

Reply:
i integrated your commit